packages/catlog/src/stdlib/theories.rs

```diff
@@ -136,6 +136,42 @@ pub fn th_category_links() -> UstrDiscreteTabTheory {
     th
 }
 
+/** The theory of stock and flow diagrams with dynamic variables.
+
+*/
+pub fn th_category_dynamic_stockflow() -> UstrDiscreteTabTheory {
+    let mut th: UstrDiscreteTabTheory = Default::default();
+    let x = ustr("Object");
+    th.add_ob_type(x);
+    // there are two proarrows from stocks to flows. they correspond to inflows and outflows.
+    th.add_mor_type(
+        ustr("Inflow"),
+        TabObType::Basic(x),
+        th.tabulator(th.hom_type(TabObType::Basic(x))),
+    );
+    th.add_mor_type(
+        ustr("Outflow"),
+        TabObType::Basic(x),
+        th.tabulator(th.hom_type(TabObType::Basic(x))),
+    );
```

Comment:
As we discussed today, these "inflows" and "outflows" are actually new kinds of links. We decided to punt on inflows/outflows until our theories can be souped up to handle terminal objects.